cmd/ci-operator-prowgen/testdata/zz_fixture_postsubmit_TestFromCIOperatorConfigToProwYaml_Using_a_variant_config__one_test_and_images__one_existing_job._Expect_one_presubmit__pre_post_submit_images_jobs._Existing_job_should_not_be_changed..yaml

```diff
@@ -70,7 +70,7 @@ postsubmits:
       volumes:
       - name: pull-secret
         secret:
-          secretName: registry-pull-credentials
+          secretName: registry-pull-credentials-all
```

Comment:
So when we're doing this, maybe we can also make the volume name and the path consistent with the secret name to make the reasoning about jobs simpler?

Reply:
Can we get the functional bit in, then rename? I want to get rid of the old secret and have `registry-pull-credentials` everywhere

Reply:
/shrug